.github/workflows/toolkit.yml

```diff
@@ -0,0 +1,36 @@
+name: toolkit
+
+on:
+  push:
+    branches: [ "3.x" ]
+  pull_request:
+    branches: [ "3.x" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+    - name: Install make
+      run:  sudo apt-get update && sudo apt-get -y install make
+    - name: Build
+      run: cd src/go; make linux-amd64; cd ../../
+      
+    - name: Build the Docker image
+      run: echo "FROM oraclelinux:9-slim" > Dockerfile; echo "COPY bin/* /usr/bin/" >> Dockerfile; docker build . --file Dockerfile --tag percona-toolkit:$(date +%s)
+    - name: Install Trivy
+      run: wget https://github.com/aquasecurity/trivy/releases/download/v0.37.1/trivy_0.37.1_Linux-64bit.deb && sudo dpkg -i ./trivy_0.37.1_Linux-64bit.deb && rm -f trivy_0.37.1_Linux-64bit.deb
+    - name: Scan Image
+      run: trivy image --scanners vuln  --format table --exit-code  1 --vuln-type  os,library --severity  CRITICAL,HIGH percona-toolkit:$(docker images | grep percona-toolkit | head -n 1 | awk '{print $2}')
```

Comment:
it might be better use github action from aqua security. you can check PSMDB operator as an example: https://github.com/percona/percona-server-mongodb-operator/blob/main/.github/workflows/scan.yml#L16-L24

Reply:
done